ccd-definition/CaseEventToFields/CareSupervision/limitedUploadDocuments.json

```diff
@@ -3,7 +3,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "limitedUploadDocuments",
-    "CaseFieldID": "otherCourtAdminDocuments",
+    "CaseFieldID": "limitedCourtAdminDocuments",
```

Comment:
This event limitedUploadDocuments is for cafcass solicitor/solicitor according to the code. What do we want to show them? Question for business I think

Reply:
cafcass solicitor/solicitor will maintain their current functionality: can add/edit/delete same documents as admin/judge (therefore present them the new Manage Documents flow). This may be wrong from business perspective, so needs further discussion